src/containers/Messages/Messages.js

```diff
@@ -19,6 +19,10 @@ const addUser = async ({ uid, email }) => {
   return database.ref('users/' + uid).set({ uid, email });
 };
 
+const addUserToken = async ({ uid, token }) => {
+  return database.ref('users/' + uid).update({ token });
+};
```

Comment:
We have the token and save usar at `verify.js` too, maybe we can do this at `verify.js`.

Reply:
Verify.js does not call every time, if the user is already logged in then we must save token if token gets removed too.